apps/mobile/src/utils/transactions.tsx

```diff
@@ -38,6 +39,10 @@ export const getTransactionType = ({ txInfo }: { txInfo: Transaction['txInfo'] }
     return ETxType.SWAP_ORDER
   }
 
+  if (isStakingTxDepositInfo(txInfo)) {
+    return ETxType.STAKE
```

Comment:
I would suggest to be more specific with this type e.g. call it `STAKE_DEPOSIT` since there can also be withdraws right?

Reply:
done